go.sum

```diff
@@ -0,0 +1,7 @@
+github.com/kr/pretty v0.1.0 h1:L/CwN0zerZDmRFUapSPitk6f+Q3+0za1rQkzVuMiMFI=
```

Comment:
mm.. when i ran `go mod tidy` i'm getting:
```
github.com/niemeyer/pretty
``` 

instead of `kr/prety`.

Reply:
I've updated it. Please have a look.